### DISCUSSION TITLE
How do I open game with mod OFF?

### DISCUSSION THREAD

**Discussion by kkzz23** [xoxor4d/p2-rtx#16] [General]
The mod is the most beautifull thing i have ever seen and I will donate you if I earn money but is there really no option to open game without loading files from the mod? Do I need to verify files on Steam to delete them every time?

**Comment by xoxor4d** [comment]
Delete or rename the following files in `root/bin`:
- `_rtx.asi`
- `d3d9.dll`
- `winmm.dll`

**Reply by kkzz23** [reply]
Thanks!

If someone wants to automate it make two .bat files in root/bin and put this into them:
For mod ON:
@echo off
:: Changing filenames RTX to ON
rename "rtx.asi" "_rtx.asi"
rename "d3d9.dll_" "d3d9.dll"
rename "winmm.dll_" "winmm.dll"
echo RTX ON.
pause

For mod OFF:
@echo off
:: Changing filenames RTX to OFF
rename "rtx.asi" "_rtx.asi"
rename "d3d9.dll" "d3d9.dll_"
rename "winmm.dll" "winmm.dll_"
echo RTX OFF.
pause
